libs/dtm/dtm_recovery.py

```diff
@@ -236,10 +237,11 @@ def process_restart(self, master_node, health_obj, pod_prefix, container_prefix,
                     return False
 
                 self.log.info("Process %s restarted successfully", process)
+            time.sleep(delay)
```

Comment:
This is intentional. Delay of 30 sec is expected between two restarts of rgw process.

Reply:
@ApurwaMohite Can we take this delay as input so that can be controlled from test,
Also this sleep can be used only when we have restart count more than 1